### PR TITLE
Client-side usability metrics

### PR DESCRIPTION
**Pull request by lucyq**

### CODE REVIEW

**Comment by lucyq** [comment]
- Pre-fill spreadsheet vs. manual data entry
- Time spent on: page, session area, table area, review/submit area, each table relevant to submission
- Validation errors
- Browser

**Comment by lucyq** [comment]
Tested & ready for merge now!

**Comment by lucyq** [comment]
@frederickjansen please merge :D

File: client/app/helper/mpc.js [hunk 1]
```diff
@@ -47,15 +48,31 @@ define([], function () {
...
+          const fields = metric[key];
+  
+          for (let f of fields) {
+            usability.push({metric: key, field: f});
+          }
+        }
+      }
+    }
+
+    return { tables, questions, usability };
```
Comment (frederickjansen) [inline client/app/helper/mpc.js]: Won't work in IE11

```suggestion
    return { tables: tables, questions: questions, usability:usability };
```

**Review by frederickjansen — CHANGES_REQUESTED** [review]

**Review by frederickjansen — APPROVED** [review]

**Pull request merged**